.github/workflows/testnet.yml

```diff
@@ -6,11 +6,13 @@ on:
     # This cron expression triggers the workflow at 8:00 UTC every day
     - cron: "8 0 * * *"
   # uncomment to debug in a PR
-  #pull_request:
+  pull_request:
```

Comment:
Intentional that you uncommented this?

Reply:
Yes, I will commit it again before merging. Just wanted to make sure that this workflow is going through